### DISCUSSION TITLE
[2.0.0] Deno.cron is not a function

### DISCUSSION THREAD

**Discussion by akomm** [denoland/deno#26296] [General]
So I've tried use `Deno.cron` but running the code that calls it with 2.0.0 I get a `not a function`.

What catches my eye is that its here under [Cloud](https://docs.deno.com/api/deno/cloud). So I thought maybe it requires some hosting context to be available. I've tried to find any mentioning of this but there is none. So I'm now not certain what is the case.

Its neither [here](https://docs.deno.com/runtime/reference/deno_namespace_apis/), nor [here](https://docs.deno.com/api/deno/), nor [here](https://docs.deno.com/api/deno/cloud), nor [here](https://docs.deno.com/api/deno/~/Deno.cron)

**Comment by marvinhagemeister** [comment]
Add the `--unstable-cron` flag or this to your `deno.json` to enable the `Deno.cron` API:

```json
{
  "unstable": ["cron"],
}
```

**Reply by akomm** [reply]
thanks

**Comment by MuellerNicolas** [comment]
This works fine, but in vscode I still get type errors, forcing me to cast it to any:

```ts
// deno-lint-ignore no-explicit-any
(Deno as any).cron("Run once a minute", "* * * * *", () => { 
    console.log("Hello, cron!");
});
```

Am I missing something or is this intended?

Thank you very much :)

**Reply by akomm** [reply]
IIRC you need to add the types and/or deno plugin in vscode (don't forget to enable).

**Reply by MuellerNicolas** [reply]
true, thank you. [enabling-deno-in-a-vs-code-workspace](https://docs.deno.com/runtime/reference/vscode/#enabling-deno-in-a-vs-code-workspace) fixed it.
